src/vs/base/browser/ui/splitview/splitview.ts

```diff
@@ -237,6 +237,9 @@ export class SplitView<TLayoutContext = undefined> extends Disposable {
 	private _onDidSashReset = this._register(new Emitter<number>());
 	readonly onDidSashReset = this._onDidSashReset.event;
 
+	private _onScroll = this._register(new Emitter<ScrollEvent>());
```

Comment:
This emitter doesn't need to exist. We can just `this.onScroll = this.scrollableElement.onScroll`, right?